results.csv

```diff
@@ -1,18 +1,33 @@
-VehicleID,Delayed in hours
```

Comment:
Die Datei ist schon in der `.gitignore`. In dieser Datei heißt es:
> results.csv
Dass die trotzdem immer wieder auftaucht bei git ist verwunderlich.